### PR TITLE
OSA - Switch from preview to GA API version

### PR DESCRIPTION
**Pull request by julienstroheker**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.4.1
+++++++
+* Using Azure RedHat Openshift GA API Version
```
Comment (amanohar) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: `Red Hat`

**Comment by julienstroheker** [comment]
Waiting after https://github.com/Azure/azure-sdk-for-python/pull/4824 to be merged

**Comment by yugangw-msft** [comment]
This is pending SDK release

**Pull request merged**